libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -31,7 +32,14 @@ export class BookSearchComponent implements OnInit {
     return this.searchForm.value.term;
   }
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    this.searchForm
```

Comment:
You will need to unsubscribe from this subscription in `ngOnDestroy` or else there will be a memory leak.

```
this.subscription = this.searchForm.get('term').valueChanges.pipe(...).subscribe(...);

// then in ngOnDestroy
this.subscription.unsubscribe();
```

You can also  move it back the effects, which is perfect fine solution as well.
```

Reply:
Unsubscribing from subscription to avoid memory leaks as per the code review comment